wasp.go

```diff
@@ -64,9 +65,11 @@ type CallResult struct {
 	Error      string        `json:"error,omitempty"`
 }
 
+type ScheduleType string
+
 const (
-	RPSScheduleType string = "rps_schedule"
-	VUScheduleType  string = "vu_schedule"
+	RPS ScheduleType = "rps_schedule"
```

Comment:
Better to change the name to something like RPU or request based?

Reply:
I still prefer `RPS` because most people don't need this corner-case mode and RPS and VU are heavily used in other load tools.